.gitlab-ci.yml

```
@@ -307,6 +307,20 @@ build:edge+flambda:
 build:base:dev:
   extends: .build-template:base:dev
 
+# Build using native dune rules
+build:base:dev:dune:
+  stage: build
+  image: $EDGE_IMAGE
```

we need
```suggestion
  image: $EDGE_IMAGE
  variables:
    OPAM_VARIANT: "+flambda"
```
when using EDGE_IMAGE